Source/OxyPlot/Series/BoxPlotItem.cs

```diff
@@ -36,12 +36,39 @@ public struct BoxPlotItem
             double upperWhisker,
             IList<double> outliers = null,
             object tag = null)
+            : this(x, lowerWhisker, boxBottom, median, double.NaN, boxTop, upperWhisker, outliers, tag)
+        {
+        }
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BoxPlotItem" /> struct.
+        /// </summary>
+        /// <param name="x">The x.</param>
+        /// <param name="lowerWhisker">The lower whisker.</param>
+        /// <param name="boxBottom">The box bottom.</param>
+        /// <param name="median">The median.</param>
+        /// <param name="mean">The mean.</param>
+        /// <param name="boxTop">The box top.</param>
+        /// <param name="upperWhisker">The upper whisker.</param>
+        /// <param name="outliers">The outliers.</param>
+        /// <param name="tag">The tag.</param>
+        public BoxPlotItem(
```

Comment:
so what would be your suggestion? just use the constructor with mean (which is a breaking change) or use the the one without mean and just use the setter for mean?

With most data types I would be on your side about not being able to change/add/remove the parameters. But a BoxPlot is defined by it's 5 base values (Lower Whisker + Box, Median, Upper Box + Whisker) and the 2 often optional values (Mean and Outliers) so if we would have to settle on one constructor (two constructors are still fine for me), I would probably go with something like this:

```c#
public BoxPlotItem(
    double x,
    double lowerWhisker,
    double boxBottom,
    double median,
    double boxTop,
    double upperWhisker)
    : this()
{
    ...
}
```

as those are the only 5 values really needed for displaying a boxplot and initialize Mean, Outliers and Tag (<-- which should definitely not be in the constructor for this data type) with the public setters.

What's your opinion on that?